packages/fxa-auth-server/lib/subhub/stubAPI.js

```diff
@@ -18,9 +18,10 @@ module.exports.buildStubAPI = function buildStubAPI(log, config) {
 
   const customer = {
     payment_type: 'credit',
-    last4: '8675',
+    last4: '7777',
```

Comment:
I confirmed with Ben `/lib/subhub/` code is essentially dead, we shouldn't need this. 😎